catalog_harvesting/cli.py

```diff
@@ -50,7 +50,11 @@ def main():
 
     if args.force_clean and args.dest:
         get_logger().info("Removing stale datasets")
-        force_clean(args.dest)
+        max_days = os.getenv('STALE_EXPIRATION_DAYS', None)
+        if max_days is not None:
+            force_clean(args.dest, max_days)
+        else:
+            force_clean(args.dest)
```

Comment:
You could shorten this to 
```
force_celan(args.dest, max_days or 3)
```

Also, STALE_EXPIRATION_DAYS is a `str` so if it is actually set this will break. You need to parse out anything that's not a string.